runtime/src/global_request_scheduler_deque.c

```diff
@@ -1,46 +1,48 @@
 #include "global_request_scheduler.h"
+#include "global_request_scheduler_deque.h"
 #include "runtime.h"
 
+#define GLOBAL_REQUEST_SCHEDULER_DEQUE_CAPACITY (1 << 19)
+
 static struct deque_sandbox *global_request_scheduler_deque;
 
 /* TODO: Should this be used???  */
 static pthread_mutex_t global_request_scheduler_deque_mutex = PTHREAD_MUTEX_INITIALIZER;
```

Comment:
Check if this is zombie code.

Reply:
Ambiguous. Opened #291 as a follow-on task. Non-blocking because this did not change in this PR.